### PR TITLE
Improved code to make segments on pipeline page responsive(#245drgx)

### PR DESCRIPTION
**Pull request by Nihu-Sharma**
### Related Issues

Closes #

### Screenshots of Visual Changes before/after (If There Are Any)

**IMPORTANT NOTICE** - Remember to add changelog entry

### Contribution and Currently Important Rules Acceptance

- [ ] I read and followed [contribution rules](https://github.com/hotwax/job-manager#contribution-guideline)

### CODE REVIEW

File: src/views/Pipeline.vue [hunk 1]
```diff
@@ -7,7 +7,7 @@
       </ion-toolbar>
 
       <ion-toolbar>
-        <div>
```
Comment (azkyakhan) [inline src/views/Pipeline.vue:10]: Remove extra space and improve indetation.

**Review by azkyakhan — CHANGES_REQUESTED** [review]

File: src/views/Pipeline.vue (hunk 1 above)
Comment (Nihu-Sharma) [reply src/views/Pipeline.vue:10]: ok ma'am

Comment (Nihu-Sharma) [reply src/views/Pipeline.vue:10]: Improved indentation and removed extra spaces.

**Comment by Nihu-Sharma** [comment]
Mobile-view-of-pipelinepage
desktop-view-of-Pipelinepage

**Review by azkyakhan — APPROVED** [review]

**Pull request merged**